js/main.js

```diff
@@ -19,9 +20,62 @@ $entryForm.addEventListener('submit', function (event) {
   data.entries.unshift(formInputValue);
   $photoPreview.src = './images/placeholder-image-square.jpg';
   $entryForm.reset();
+  var list = renderEntryDomTree(formInputValue);
+  entryHolder.prepend(list);
+  changeView('entries');
 });
 
-window.addEventListener('beforeunload', function () {
-  var formStorage = JSON.stringify(data);
-  localStorage.setItem('code-journal-input', formStorage);
+function renderEntryDomTree(entry) {
+  var descriptionEntry = document.createElement('p');
+  var descriptionText = document.createTextNode(entry.notes);
+  descriptionEntry.appendChild(descriptionText);
+
+  var headingEntry = document.createElement('h2');
+  var headingText = document.createTextNode(entry.title);
+  var elementI = document.createElement('i');
+  elementI.setAttribute('class', 'fas fa-pencil-alt');
+  headingEntry.append(headingText, elementI);
+
+  var divInputSection = document.createElement('div');
+  divInputSection.setAttribute('class', 'column-half input-section');
+
+  divInputSection.append(headingEntry, descriptionEntry);
+
+  var imgEntry = document.createElement('img');
+  imgEntry.src = entry.photoURL;
+
+  var divImageSection = document.createElement('div');
+  divImageSection.setAttribute('class', 'column-half image-section');
+
+  divImageSection.append(imgEntry);
+
+  var liAnEntry = document.createElement('li');
+  liAnEntry.setAttribute('class', 'row an-entry');
+  liAnEntry.setAttribute('data-entry-id', entry.entryId);
+
+  liAnEntry.append(divImageSection, divInputSection);
+
+  return liAnEntry;
+}
+window.addEventListener('DOMContentLoaded', function () {
+  for (var ent = 0; ent < data.entries.length; ent++) {
+    var list = renderEntryDomTree(data.entries[ent]);
+    entryHolder.appendChild(list);
+  }
 });
+
+var buttonNewEntry = document.getElementById('new-entry');
+buttonNewEntry.addEventListener('click', function () {
+  changeView('entry-form');
+});
+
+function changeView(view) {
+  if (view === 'entry-form') {
+    document.querySelector('[data-view=entry-form]').className = '';
+    document.querySelector('[data-view=entries]').className = 'hidden';
+  } else {
+    document.querySelector('[data-view=entry-form]').className = 'hidden';
+    document.querySelector('[data-view=entries]').className = 'container';
+  }
+}
+changeView('entries');
```

Comment:
This will make it so that the user is always shown the `entries` view first. You want to show the same view they were on right before the page closed last time they visited the site. You should be updating the value of `data.view` to be the name of the current view every time the view changes. Once you do that, you can check the value of `data.view` when the page loads, and show that view to the user.